recipes/iqtree/meta.yaml

```diff
@@ -0,0 +1,26 @@
+{% set version = "1.5.2" %}
+
+package:
+  name: "iqtree"
+  version: "{{ version }}"
+
+build:
+    number: 0
+
+source:
+  fn: iqtree-{{ version }}-Linux-x86_64.tar.gz # [linux]
+  url: https://github.com/Cibiv/IQ-TREE/releases/download/v1.5.2/iqtree-omp-{{ version }}-Linux.tar.gz # [linux]
+  md5: c62423f67ef4b21af46768c4cb5a02c0 # [linux]
```

Comment:
This is because many of us run bioconda tools on HPC environment and precompiled binaries from somewhere can work on old HPC systems, but usually they don't. This is our way to guarantee that these binaries are relocatable.